cot/src/router.rs

```diff
@@ -121,7 +126,7 @@ impl Router {
     /// Get a URL for a view by name.
     ///
     /// Instead of using this method directly, consider using the
-    /// [`reverse!`](crate::reverse) macro which provides much more
+    /// [`reverse!`](crate::reverse_redirect) macro which provides much more
```

Comment:
Shouldn't it be?
```suggestion
    /// [`reverse_redirect!`](crate::reverse_redirect) macro which provides much more
```

Reply:
Should actually be `[`reverse!`](crate::reverse)`, thanks for catching this!